utils/Constants.ts

```diff
@@ -1,11 +1,13 @@
 // Spotify app client ID
-export const ClientId = '7fca87b90c354988acbffcb363ed7a60';
+export const ClientId = process.env.CLIENT_ID;
```

Comment:
Can you please add these new env variables to the README? Under `#deploying-own-vercel-instance` and `#running-locally`. Thanks for making this change!

Reply:
I think it's already there. It's the `CLIENT_ID`. I just use the `.env` variables, which don't work currently. I keep getting `undefined` for everything in the `process.env.`

Reply:
I meant `BASE_URL` and `REDIRECT_URI`

Reply:
Thank you :)